lms/static/js/instructor_dashboard/open_response_assessment.js

```diff
@@ -0,0 +1,229 @@
+/* globals _ */
+
+(function(Backbone, $, _, gettext) {
```

Comment:
I'd love ORA code to stay in the edx-ora2 repo. Why does this code need to be here instead of exposing a view defined in edx-ora2?